plugin/evm/header/gas_limit.go

```diff
@@ -9,12 +9,17 @@ import (
 
 	"github.com/ava-labs/avalanchego/utils/math"
 	"github.com/ava-labs/coreth/core/types"
-	"github.com/ava-labs/coreth/params"
 	"github.com/ava-labs/coreth/params/extras"
 	"github.com/ava-labs/coreth/plugin/evm/upgrade/ap1"
 	"github.com/ava-labs/coreth/plugin/evm/upgrade/cortina"
 )
 
+const (
+	MinGasLimit          uint64 = 5000               // Minimum the gas limit may ever be.
+	MaxGasLimit          uint64 = 0x7fffffffffffffff // Maximum the gas limit (2^63-1).
+	GasLimitBoundDivisor uint64 = 1024               // The bound divisor of the gas limit, used in update calculations.
+)
```

Comment:
Similar comment as the other file, these aren't constants that apply generally to the header, they are specific to `ap0`

Also:
```suggestion
const (
	MinGasLimit          = 5000               // Minimum the gas limit may ever be.
	MaxGasLimit          = 0x7fffffffffffffff // Maximum the gas limit (2^63-1).
	GasLimitBoundDivisor = 1024               // The bound divisor of the gas limit, used in update calculations.
)
```

Reply:
moving all them to ap0